packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -613,8 +613,8 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
       .toList();
   }
 
-  int? search(List<DropdownMenuEntry<T>> entries, TextEditingController textEditingController) {
-    final String searchText = textEditingController.value.text.toLowerCase();
+  int? search(List<DropdownMenuEntry<T>> entries) {
+    final String searchText = _localTextEditingController!.value.text.toLowerCase();
```

Comment:
@rkishan516 should these changes be here? They appear unrelated.

Reply:
Sorry, I don't know, How I messed it up today. I have fixed and pushed.